src/ui/UserInterface.java

```diff
@@ -76,7 +76,7 @@ public static UserInterface instance() {
 	 * @param errorMessage - a custom error message in case of invalid input
 	 * @return String entered by user without leading and trailing spaces
 	 */
-	public static String getString(String prompt, String errorMessage) {
+	public static String getString(String prompt) {
```

Comment:
Could you explain why we don't want this error message in the parameter?

Reply:
Yes! I figured that it's going to be the same message across the board in GroceryStore (the methods are private) and I coded a constant for all instances (which will only happen when the user does not respond but rather just hits enter (response is "")) and I coded it into the GroceryStore class and used it across all getString method occurrences. Consequently, the getString method is used by all the "getXXXXX" helper methods to retrieve an answer, always having the same error message, if just <enter> was pressed WITHOUT ANY typed input.

Reply:
So this method still accounts for the user entering something else like "sure" into the prompt?

Reply:
If it's a yes/no question as handled by getYesOrNo() method, it will be resolved as an error. getYesOrNo() will first cal getString(), which will eliminate a "" answer, then continue with filtering the first letter no only "y/Y" or "n/N".